### DISCUSSION TITLE
Turn blacklist

### DISCUSSION THREAD

**Discussion by Eirikr70** [processone/ejabberd#4231] [Questions] [answered]
Hello,
Before setting up an ejabberd server (as a docker container), I used to operate a matrix/synapse/coturn server.
In order to secure the coturn server, there is a standard list of denied peer ip's, which seems similar to the turn blacklist in ejabberd. The standard denied peer ip's list is as follows (according to [https://matrix-org.github.io/synapse/v1.56/turn-howto.html](https://matrix-org.github.io/synapse/v1.56/turn-howto.html))
- 10.0.0.0-10.255.255.255
- 192.168.0.0-192.168.255.255
- 172.16.0.0-172.31.255.255

Since I am far from being a network expert, I have initially considered that a similar list should be put in place as my turn blacklist, but then the turn server didn't work properly.

Is there any standard blacklist for excluding internal IP addresses for security ?

**Comment by weiss** [comment, marked as answer]
You could check the list [suggested by Enable Security][1].  However, the list of address ranges that can safely be blocked depends on your environment.  E.g., if your server is running within a local network which is supposed to be able to use TURN, that network's address range must obviously be whitelisted. 

[1]: https://rtcsec.com/article/cve-2020-26262-bypass-of-coturns-access-control-protection/#further-concerns-what-else

**Comment by C0la149** [comment]
I apologize for being off topic. But can you contact me? my email is aryfe53@gmail.com 
@Eirikr70 

I have a problem with nextcloud. And you were on topic and solved the problem. 

[https://help.nextcloud.com/t/setting-up-cron-something-has-gone-wrong/127215](url)

Can you help me?(
